### PR TITLE
Add Monika ghost intro and Natsuki desk art

### PR DESCRIPTION
**Pull request by kaido1224**
Natsuki desk art made by Iron-707

# TESTING
1. add `return store.mas_getEV("greeting_ghost")` to the first line in the `selectGreeting` function in `script-greetings`
2. Launch game and get scared

Unlocking:
- answering Yes to the spoops question will unlock the gret
- the greeting should lock itself
- there is an update script, but we can test that later.

### CODE REVIEW

File: Monika After Story/game/script-easter-eggs.rpy [hunk 1]
```diff
@@ -231,3 +231,33 @@ label natsuki_name_scare_hungry:
...
+label mas_ghost_monika:
+    python:
+        #plays music from ghost menu.
+        play_song(audio.ghostmenu)
+        
+    #show ghost monika
+    show ghost_monika zorder MAS_MONIKA_Z at i11
+    
+    #wait 10 seconds (length of ghost menu music)
+    pause(10)
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-easter-eggs.rpy]: switch this to a hard pause so users cant just click through this
```suggestion
    $ renpy.pause(10, hard=True)
```

**Comment by kaido1224** [comment]
I'll talk to aldo tomorrow about what changes i need to make to make it good for the new greeting system.

File: Monika After Story/game/script-greetings.rpy [hunk 2]
```diff
@@ -3324,3 +3324,23 @@ label ch30_reload_delegate:
         call expression reload_label
 
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.greeting_database,
+            eventlabel="greeting_ghost",
+            unlocked=False,
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-greetings.rpy]: since unlocked is False here, it wont trigger. Are you planning to unlock this in a future topic?

Comment (jmwall24) [reply Monika After Story/game/script-greetings.rpy]: Probably needs to be unlocked by/update scripted for do you like spooks

**Pull request merged**